pdpserver/server/stream_service.go

```diff
@@ -55,7 +55,11 @@ func (s *Server) NewValidationStream(stream pb.PDP_NewValidationStreamServer) er
 				return buffer, nil
 			})})
 		} else {
-			err = stream.Send(&pb.Msg{Body: s.rawValidateToBuffer(p, c, in.Body, buffer)})
+			body, err := s.rawValidateToBuffer(p, c, in.Body, buffer)
```

Comment:
Here you define new `err` variable, which shadows existing one. Can you make sure that this doesn't break existing logic, e.g. now external `err` is not updated with the error coming from `Send`. Consider renaming the new error variable to avoid ambiguity.